apps/ledger-live-desktop/src/renderer/families/stacks/MemoValueField.jsx

```diff
@@ -0,0 +1,50 @@
+// @flow
+
+import React, { useCallback } from "react";
+import { getAccountBridge } from "@ledgerhq/live-common/bridge/index";
+import Input from "~/renderer/components/Input";
+import invariant from "invariant";
+import type { Account } from "@ledgerhq/types-live";
+import type { Transaction, TransactionStatus } from "@ledgerhq/live-common/generated/types";
+import {useTranslation} from "react-i18next";
+
+const MemoValueField = ({
+  onChange,
+  account,
+  transaction,
+  status,
+}: {
+  onChange: string => void,
+  account: Account,
+  transaction: Transaction,
+  status: TransactionStatus,
+}) => {
+  invariant(transaction.family === "stacks", "MemoField: stacks family expected");
+
+  const { t } = useTranslation();
+
+  const bridge = getAccountBridge(account);
+
+  const onMemoValueChange = useCallback(
+    memoValue => {
+      onChange(bridge.updateTransaction(transaction, { memo: memoValue }));
+    },
+    [onChange, transaction, bridge],
+  );
+
+  // We use transaction as an error here.
+  // It will be usefull to block a memo wrong format
```

Comment:
shouldn't it be named as `.memo`? (as it maps to "memo" field, essentially the name of the subfields are opened to the implementation but it's good if we semantically maps it to fields)

Reply:
Yes, it could be. However, if we need to define some specific types for memo in the future, it will be easier in this way. I could be renamed though.